clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/ArraysExtensions.scala

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.clio.transfer.model.arrays
+
+/**
+  *  Extensions for files in the arrays index.
+  */
+object ArraysExtensions {
+
+  val BpmExtension = ".bpm"
+
+  val CsvExtension = ".csv"
+
+  val DictExtension = ".dict"
+
+  val EgtExtension = ".egt"
+
+  val FastaExtension = ".fasta"
+
+  val FaiExtension = ".fai"
+
+  val GzExtension = ".gz"
+
+  val IdatsExtension = ".idats"
```

Comment:
should be idat extension.  There are two idat files that we need to store here.  The workflow uses the idats file, but it's contents are the paths to the red and green idats.  We don't need to store anything about the idats file if you get what I mean.

Reply:
Not sure I get what you mean about which files to store. I was purposely redundant here to make it easier to just delete what we don't want rather than have to discover what we do want. We obviously don't have to store both the file containing the idat paths and the idat paths themselves. I think it would be simpler to just store the two .idat files, but want confirmation.